pkg/cmd/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -143,10 +144,13 @@ func (o *ReconcileClusterRoleBindingsOptions) Validate() error {
 	return nil
 }
 
-// ReconcileClusterRoleBindingsOptions contains all the necessary functionality for the OpenShift cli reconcile-cluster-role-bindings command
 func (o *ReconcileClusterRoleBindingsOptions) RunReconcileClusterRoleBindings(cmd *cobra.Command, f *clientcmd.Factory) error {
 	changedClusterRoleBindings, err := o.ChangedClusterRoleBindings()
-	if err != nil {
+	if IsClusterRoleBindingLookupError(err) {
```

Comment:
Also return the aggregate of the replace (down below) and this if its non-nil.

The basic idea is that we should take the action we can.  If you want to argue it, I could see skipping the Replace, but not the display.